src/components/landing/layout/version-info/version-info.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { Fragment, useState } from 'react'
+import { Button, Col, Modal, Row } from 'react-bootstrap'
+import { Trans, useTranslation } from 'react-i18next'
+import { useSelector } from 'react-redux'
+import { Link } from 'react-router-dom'
+import { ApplicationState } from '../../../../redux'
+import frontendVersion from '../../../../version.json'
+import { VersionInputField } from './version-input-field'
+
+export const VersionInfo: React.FC = () => {
+  const [show, setShow] = useState(false)
+
+  const handleClose = () => setShow(false)
+  const handleShow = () => setShow(true)
+
+  const { t } = useTranslation()
+
+  const serverVersion = useSelector((state:ApplicationState) => state.backendConfig.version)
+
+  const column = (title: string, version: string, sourceCodeLink: string, issueTrackerLink: string) => (
+    <Col md={6} className={'flex-column'}>
+      <h5>{title}</h5>
+      <VersionInputField version={version}/>
+      { sourceCodeLink
+        ? <a target="_blank"
+          rel="noopener noreferrer"
+          href={sourceCodeLink}
+          className={'btn btn-sm btn-primary d-block mb-2'}>
+          <Trans i18nKey={'sourceCode'}/>
+        </a> : null }
+      { issueTrackerLink
+        ? <a target="_blank"
+          rel="noopener noreferrer"
+          href={issueTrackerLink}
+          className={'btn btn-sm btn-primary d-block'}>
+          <Trans i18nKey={'issueTracker'}/>
```

Comment:
better use translated-extranal-link?

Reply:
The translated external link isn't a button. And I don't know if its overkill to add this.

Reply:
Well. Currently the external link and the translated external link are special components for the footer. But we could make them more general :thinking:

Reply:
I think it's okay to leave it at is, if we say external-links and tranlsated-external-links are footer only

Reply:
done